frontend/main-app/src/components/FormViewer/FormViewer.tsx

```diff
@@ -38,29 +51,47 @@ export const FormViewer = (): React.JSX.Element => {
     }
   };
 
-  const RenderForm = () => {
-    return (
-      <main className={classes.card}>
-        <form onSubmit={handleSubmit}>
-          <Heading level={1} size="xlarge" spacing>
-            {formSchema.title}
-          </Heading>
-          {formSchema.components.map((component) => (
-            <span key={component.name} className={classes.component}>
-              <FormComponent component={component} error={formErrors[component.name]} />
-            </span>
-          ))}
-          <div className={classes.buttonContainer}>
-            <Button type="submit" size={"large"} fullWidth={false}>
-              {t("form_viewer.submit")}
-              <TasklistSendFillIcon />
-            </Button>
+  const RenderForm = () => (
+    <main className={classes.card}>
+      <form onSubmit={handleSubmit}>
+        <Heading level={1} size="xlarge">
+          {formSchema.title}
+        </Heading>
+        {formSchema.components.map((component) => (
+          <div key={component.name} className={classes.component}>
+            <FormComponent component={component} error={formErrors[component.name]} />
           </div>
-        </form>
-        {alertToRender(formAlert, t)}
-      </main>
-    );
-  };
+        ))}
+        <div className={classes.buttonContainer}>
+          <Button type="submit" size={"large"} fullWidth={false}>
+            {t("form_viewer.submit")}
+            <TasklistSendFillIcon />
+          </Button>
+        </div>
+      </form>
+      {alertToRender(formAlert, t)}
+    </main>
+  );
+
+  const RenderDraft = () => (
+    <main className={classes.card}>
+      <Heading level={1} size="xlarge" spacing>
+        {formSchema.title}
+      </Heading>
+      <Paragraph>
+        <Trans i18nKey="form_viewer.draft" />
+      </Paragraph>
+      <Paragraph>
+        <Link href={formBuilderLink}>{t("form_viewer.form_builder.link")}</Link>
+      </Paragraph>
+    </main>
+  );
 
-  return formSchema ? <RenderForm /> : <NotFound />;
+  return (
+    <>
+      {formSchema && !isDraft && <RenderForm />}
+      {formSchema && isDraft && <RenderDraft />}
+      {!formSchema && <NotFound />}
+    </>
```

Comment:
```suggestion
  return (
    <>
      {formSchema && isDraft ? <RenderDraft /> :  <RenderForm />}
      {!formSchema && <NotFound />}
    </>
```

Nitpicking, but I think this is a better approach here, what do you think?

Reply:
I agree 👍